docs/nightly/en/user-guide/operations/configuration.md

```diff
@@ -150,432 +31,26 @@ For example, start the standalone as below:
 greptime standalone start -c standalone.example.toml
 ```
 
-### Protocol options
```

Comment:
No, I don't think we can remove these sections. Because the `config.md` is mostly like a reference, but not a user guide. 

These sections describe configuration structures and semantics to show how configurations are organized and guide changes.

We should always differentiate user guides and references.